packages/chat-component-bindings/src/messageThreadSelector.ts

```diff
@@ -108,6 +108,7 @@ export const messageThreadSelector = createSelector(
             message.type === ACSKnownMessageType.topicUpdated ||
             message.clientMessageId !== undefined
         )
+        .filter((message) => message.content && message.content.message !== '') // TODO: deal with deleted message and remove
```

Comment:
Curious, what would "dealing with a deleted message" require?

Reply:
We need 2 materials:
1. we need to pump isDeleted property from chatClient to stateful client
2. We need another component to show deleted message like this:
![image](https://user-images.githubusercontent.com/11863655/131760308-d85121d5-6178-4c52-833a-a28f235fddb6.png)

Reply:
Gotcha. Thanks